<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HISTClass</class>
 <widget class="QWidget" name="HISTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HIST</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>200</height>
            </size>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideLeft</enum>
           </property>
           <property name="rowCount">
            <number>1</number>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <row/>
           <column>
            <property name="text">
             <string>ファイル名</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string> </string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>302</width>
           <height>239</height>
          </size>
         </property>
         <property name="text">
          <string>画像描画領域</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QFrame" name="frame_2">
                 <property name="minimumSize">
                  <size>
                   <width>560</width>
                   <height>235</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <widget class="QLabel" name="label_6">
                  <property name="geometry">
                   <rect>
                    <x>300</x>
                    <y>210</y>
                    <width>30</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>150</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_7">
                  <property name="geometry">
                   <rect>
                    <x>200</x>
                    <y>210</y>
                    <width>30</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>100</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_3">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>210</y>
                    <width>16</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_4">
                  <property name="geometry">
                   <rect>
                    <x>510</x>
                    <y>210</y>
                    <width>30</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>255</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_5">
                  <property name="geometry">
                   <rect>
                    <x>400</x>
                    <y>210</y>
                    <width>30</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>200</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_8">
                  <property name="geometry">
                   <rect>
                    <x>100</x>
                    <y>210</y>
                    <width>30</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>50</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="histgramRGB">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>512</width>
                    <height>200</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>52</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="text">
                   <string>グラフ描画領域</string>
                  </property>
                 </widget>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>10</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QFrame" name="frame_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>560</width>
                   <height>235</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <widget class="QLabel" name="label_9">
                  <property name="geometry">
                   <rect>
                    <x>270</x>
                    <y>210</y>
                    <width>30</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>180</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_11">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>210</y>
                    <width>16</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_12">
                  <property name="geometry">
                   <rect>
                    <x>510</x>
                    <y>210</y>
                    <width>30</width>
                    <height>21</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>360</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="histgramH">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>512</width>
                    <height>200</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>52</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="text">
                   <string>グラフ描画領域</string>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hist.qrc"/>
 </resources>
 <connections/>
</ui>
